plugins/modules/vmware_guest.py

```diff
@@ -3231,20 +3274,29 @@ def reconfigure_vm(self):
                 uuid_action_opt.value = "create"
                 self.configspec.extraConfig.append(uuid_action_opt)
 
+            # TODO latent bug? self.change_detected isn't checked again after this
```

Comment:
should TODO be addressed?

Reply:
I think you're right, it isn't checked later so I'll remove it.